.github/workflows/CODE_STANDARDS.md

```diff
@@ -357,6 +357,8 @@ Good example:
 feat: add possibility to create a new user by admin (#BUX-123)
 ```
 
+<!--- # How to rename the BUX task name -->
+
```

Comment:
How to rename the `BUX` task name to out new naming convention?

Reply:
I've changed to `SPV Wallet`

Reply:
Like this?:
```
feat: add possibility to create a new user by admin (#123)
```

Reply:
rather like 
`feat(#123): add possibility to create a new user by admin`